lib/intl.ts

```diff
@@ -199,6 +199,15 @@ if ('formatRangeToParts' in IntlDateTimeFormat.prototype) {
 
 DateTimeFormatImpl.prototype = Object.create(IntlDateTimeFormat.prototype, properties);
 
+// Ensure that the prototype isn't writeable.
+Object.defineProperty(DateTimeFormatImpl, 'prototype', {
+  writable: false,
+  enumerable: false,
+  configurable: false
```

Comment:
We could consider merging this with line 200 and adding `value: Object.create(IntlDateTimeFormat.prototype, properties)` but I don't know if TypeScript is smart enough to see through that...

Reply:
I tried that earlier and got errors saying that a read-only prototype couldn't be overwritten. This code was the fallback solution. :-(